tools/checkcomments/checkcomments.go

```diff
@@ -53,10 +56,11 @@ func main() {
 
 // run analyses TODO comments.
 func run(pass *analysis.Pass) (any, error) {
-	var client *client
+	var client *github.Client
+	issueReference := "// TODO"
```

Comment:
I think making it a named variable makes it less clear: what is "issue reference"? why is it a variable?